src/main/java/io/phdata/streamliner/schemadefiner/attributeretriever/NoAttributeRetriever.java

```diff
@@ -0,0 +1,49 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+package io.phdata.streamliner.schemadefiner.attributeretriever;
+
+import io.phdata.streamliner.schemadefiner.model.FileFormat;
+import java.util.List;
+import java.util.Map;
+
+public class NoAttributeRetriever implements AttributeRetriever {
+
+  private final String dbType;
+
+  public NoAttributeRetriever(String dbType) {
+    this.dbType = dbType;
+  }
+
+  @Override
+  public List<Map<String, Object>> retrieve(String schema, String tableName) {
+    throw new RuntimeException(
+        String.format(
```

Comment:
This can throw a `NotImplementedException` instead of plain `RuntimeException`, it conveys a little more information